tests/hedgehog_test_runner.py

```diff
@@ -173,13 +174,19 @@ def update_allure_report():
             layer_label["value"] = tests_to_grup_map[test_info["fullName"]]
         else:
             layer_label["value"] ="Default"
+            unmapped_tests.append(test_info["fullName"])
 
         test_info["labels"].append(layer_label)
 
         #rewrite updated JSON
         with open(file, "w") as outfile:
             json.dump(test_info, outfile)
 
+        #write list with unmapped tests to file
+        if unmapped_tests:
+            with open(FULL_REPORT_DIR_PATH +'/unmapped_tests.txt', 'w') as outfile:
```

Comment:
We can save `unmapped_tests.txt` as separate field into **_vsTestbed-01-t0.yaml_**, need to discuss it